### ISSUE TITLE
showColoredLineSymbols: true - doesn't work

### ISSUE THREAD

**Issue by djsunrise19** [Corburn84/MMM-PublicTransportDB_modified#1]
Hi,

I asked that question in the original repository before, but nobody answered / helped. I have noticed that the `showColoredLineSymbols: true` option doesn't work. `false` is working (grey background), true isn't. The product types seems to be undefined. Do you have an idea?

I tried to fix that, but I haven't so much experience and can't find the problem.

![](https://user-images.githubusercontent.com/14234570/36032768-52ce522c-0daf-11e8-9266-5e26eef926b6.png)

**Comment by Corburn84** [comment]
Hey,

thx for the hint. Didn't notice the missing color.
Well, "color" is defined but has the value #000000 = black.
The bug is in DBFetcher.js:

`type: productType.type`

This is wrong and needs to be

`type: row.line.product`.

I gonna fix that.

**Comment by djsunrise19** [comment]
Ah, great. Thank you. But now, there seems to be an overlay?!
![image](https://user-images.githubusercontent.com/14234570/36110462-f4e81780-1022-11e8-86c4-f0df6687fa84.png)

**Comment by Corburn84** [comment]
Yeah, I noticed as well. The cell of the table in the wrapper is too short if it gets filled with a background. Dunno why  😄 we gonna find out, might just be a little thingy in the style.css or something like that.

**Comment by Corburn84** [comment]
Fixed the style.css.
Added nowrap option for the whole table (just in case any other type leads to word wrap as well) and increased size of .dbsign from 2.5em to 4.5em.

![grafik](https://user-images.githubusercontent.com/9165559/36168826-a7b4b4ae-10fa-11e8-9585-c8493ffae647.png)

I might gonna add some more space to the direction cell, so that the direction text won't be that close to the "RB ..." cell.
